modules/block/src/main/kotlin/jp/co/soramitsu/iroha2/Genesis.kt

```diff
@@ -27,6 +28,17 @@ open class Genesis(open val genesisBlock: RawGenesisBlock) {
     fun asJson(): String = JSON_SERDE.writeValueAsString(this.genesisBlock)
 
     companion object {
+        /**
+         * List of genesis blocks to single block with unique instructions
+         */
+        fun List<Genesis>.toSingle(): Genesis {
+            val uniqueIsi: MutableSet<Instruction> = mutableSetOf()
```

Comment:
aren't there any other metadata included to the instructions? can we be sure there are no duplicated instructions from the logic perspective? (like creating the same accounts)